src/ralph/parsers.py

```diff
@@ -0,0 +1,82 @@
+"""
+Ralph tracking logs parsers.
+"""
+
+import logging
+import os
+
+import pandas as pd
+
+logger = logging.getLogger(__name__)
+
+
+class BaseParser:
+    """Base tracking logs parser."""
+
+    def parse(self, input_file, filters=None, chunksize=1):
+        """Parse GELF formatted logs (one json string event per row).
+
+        Args:
+            input_file (string): Path to the log file to parse.
+            filters (list): A list of functions to apply on parsed results to
+                filter them.
+            chunksize (int): The amount of log records to process at a time.
+
+        Returns:
+            Parser implementation should return a pandas DataFrame object with
```

Comment:
The return value of the BaseParser should be a generator, to be consistent with GELFParser ?